com.microsoft.mrtk.spatialmanipulation/Solvers/TapToPlace.cs

```diff
@@ -467,19 +493,22 @@ protected virtual void SetRotation()
         /// </summary>
         private void RegisterPlacementAction()
         {
-            InputAction placementAction = GetInputActionFromReference(placementActionReference);
-            if (placementAction == null)
+            foreach(InputAction placementAction in placementActions)
             {
-                Debug.Log("Failed to register the placement action, the action reference was null or contained no action.");
-                return;
+                if (placementAction == null)
+                {
+                    Debug.Log("Failed to register the placement action, the action reference was null or contained no action.");
+                    return;
```

Comment:
We shouldn't just return here. That could lead to some unexpected behavior. Like lastActions not getting set or non-null performed events not being unregistered later on. So change `return` to `continue`